### ISSUE TITLE
State of the project

### ISSUE THREAD

**Issue by Kononnable** [Kononnable/typeorm-model-generator#279]
Unfortunately for last few weeks I didn't have enough free time to be able to spend it on this project(or OSS in general). I just wanted you all to know that this project isn't abandoned and I plan to get back to the issues/PRs once I got some more time(but I don't have a clue when this is gonna happen, might be week, might be month etc.). 
I think typeorm-model-generator is generally in a good condition, all commonly requested features are there or there is a easy workaround if something isn't working as expected. There are always some things which can be improved. If you encounter such cases for now I suggest customizing behavior using [this](https://github.com/Kononnable/typeorm-model-generator/blob/master/USECASES.md#clone-repo-and-make-manual-changes) approach. Customizing behavior for your specific use case(without carrying about compatibility with all available options) should be relatively simple in most of the cases.
You can still open new issues/PRs, but be aware that it might take some before I review them.

**tl;dr** Currently I don't have time for this project, I plan to get back to it in the future.
